pkgs/servers/web-apps/kasmweb/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, lib
+, fetchzip
+, yq-go
+}:
+stdenv.mkDerivation rec {
+  pname = "kasmweb";
+  version = "1.12.0";
+  build = "d4fd8a";
+
+  src = fetchzip {
+    url = "https://kasm-static-content.s3.amazonaws.com/kasm_release_${version}.${build}.tar.gz";
+    sha256 = "sha256-dCjWmI8gYtoMiMHVNgTg2ZROHXvT4ulynNvnKfMxURo=";
+  };
+
+  nativeBuildInputs = [ yq-go ];
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir $out
+    rm bin/utils/yq*
+    cp -r bin conf www $out/
+  '';
+
+
+  meta = with lib; {
+    homepage = "https://www.kasmweb.com/";
+    description = "Streaming containerized apps and desktops to end-users. The Workspaces platform provides enterprise-class orchestration, data loss prevention, and web streaming technology to enable the delivery of containerized workloads to your browser.";
```

Comment:
Short description should have just one phrase. If you feel the need of bigger descriptions, use `longDescription`.

(And break `longDescription` in the 80-char limit.)

Reply:
Sure. Will fix.